tac/meeting-minutes/2025/2025-02-20.md

```diff
@@ -0,0 +1,62 @@
+- The [LF Decentralized Trust /dev/weekly developer newsletter](https://lf-hyperledger.atlassian.net/wiki/spaces/DR/pages/17170445/dev+weekly+Newsletter) goes out each Friday to hundreds of LF Decentralized Trust developers. It is a collaborative effort. If you have a project release, pull request, community event, and/or relevant article you would like highlighted next week, please [leave a comment for consideration on the upcoming newsletter wiki page](https://lf-hyperledger.atlassian.net/wiki/spaces/DR/pages/75268141/2025).
+- The annual LFDT mentorship program is now accepting project proposals from people who are looking to mentor people who want to work on your project or lab.  This is a great opportunity to bring more contributors into the community who can help you.  Submissions for project proposals are now open and will close on March 15.  Note that the submission form has been moved from Confluence to Github so this will look different for people who have submitted proposals in the past.  You can submit project proposals at: [LF-Decentralized-Trust-Mentorships/mentorship-program](https://github.com/LF-Decentralized-Trust-Mentorships/mentorship-program/issues/new?template=mentorship-project.yml)
+- Weds March 26th @ 8amPT, join the Besu community for the LF Decentralized Trust workshop, “Configure, Deploy, and Manage a Besu Private Network.” If you are looking for a solid starting point to become a Besu Certified Professional, this free workshop should be on your calendar. [Register here](https://zoom.us/webinar/register/8117379999563/WN_NZ8FgWbdThiDp_8TCByjkQ). [Find out more on the wiki](https://lf-hyperledger.atlassian.net/wiki/spaces/events/pages/92372993/Configure+Deploy+and+Manage+a+Besu+Private+Network).
+- Please review [Project Smoot Proposal](https://github.com/LF-Decentralized-Trust/project-proposals/pull/24) for review in next week's TAC meeting
+
+# Quarterly reports
+- [2024 Q4 Besu](https://github.com/LF-Decentralized-Trust/governance/pull/92)
+- [2024 Q4 Hyperledger Solang (due November 21, 2024)](https://github.com/LF-Decentralized-Trust/governance/pull/96)
+
+# Annual reports
+- [Hyperledger Cacti Annual Report](https://github.com/LF-Decentralized-Trust/governance/pull/108) (Arun and Enrique)
+- [Hyperledger Indy Annual Report](https://github.com/LF-Decentralized-Trust/governance/pull/104) (Enrique and Peter) - to discuss in February 20th meeting
+- [Hyperledger Iroha](https://github.com/LF-Decentralized-Trust/governance/pull/109) (Matthew and Marcus) - to discuss in February 27th meeting
+- [Hyperledger AnonCreds Annual Report](https://github.com/LF-Decentralized-Trust/governance/pull/98) (Diane and Rama) - to discuss in March 6th meeting
+- [Web3j Annual Report](https://github.com/LF-Decentralized-Trust/governance/pull/112) (Marcus and Char)
+- [Hyperledger Bevel](https://github.com/LF-Decentralized-Trust/governance/pull/113) (Marcus and Wenjing)
+
+# Overdue reports
+- Hyperledger Solang (due February 13, 2025)
```

Comment:
Solang report found in https://github.com/LF-Decentralized-Trust/governance/pull/96